src/app/containers/PageHandlers/withVariant/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useParams, useLocation, Redirect } from 'react-router-dom';
+import getVariantRedirectUrl from './getVariantRedirectUrl';
+
+const WithVariant = Component => {
+  const VariantContainer = props => {
+    const { service, variant } = useParams();
```

Comment:
Can you have this return `<Component {...props} />` `if (!onClient) {` please?

Reply:
Fixed!